runtime/src/bank.rs

```diff
@@ -5180,9 +5237,9 @@ impl Bank {
             &self.rc.accounts.accounts_db,
             &self.ancestors,
             sanitized_txs,
-            check_results,
+            check_results.to_owned(),
```

Comment:
We should probably change `load_accounts()` to take `lock_results: &[TransactionCheckResult],` as the argument. That'll remove the need for `to_owned()`. That will require some changes to `NonceFull`. Maybe do that as a separate PR first?